nemo/collections/nlp/parts/nlp_overrides.py

```diff
@@ -115,7 +115,7 @@ def configure_ddp(self):
             hasattr(self.model, 'with_distributed_adam') and self.model.with_distributed_adam
         ):
             # do not use DDP if using megatron amp O2 or distributed optimizer
-            self._model = LightningDistributedModule(self.model)
+            self._model = _LightningModuleWrapperBase(self.model)
```

Comment:
LGTM. Note that in in the future 2.1 version, this minor annoyance of having to wrap the module with our special wrapper won't be necessary anymore 🎉